inst/extdata/LICENSE_data.txt

```diff
@@ -0,0 +1,6 @@
+Files:
+- publishedweek392020.xlsx
+- covid19infectionsurveydatasets20200528.xlsx
+- covid19infectionsurveydatasets20201009.xlsx
```

Comment:
I've given links to all of these datasets, both the landing page and the specific version downloaded. I think we should keep these specific versions because the serosurveys are cumulative and don't have the time series (hence the inclusion of two to get timepoints), and because the data used is specific to certain dates, as noted in the vignette. So we don't want to update any of them.